locale/zh-cn/about/index.md

```diff
@@ -25,23 +25,20 @@ server.listen(port, hostname, () => {
 });
 ```
 
-这与今天使用 OS 线程的更常见并发模型形成了对比。基于线程的网络效率相对低下，使用起来非常困难。此外，Node.js 的用户不必担心死锁过程，因为没有锁。Node 中几乎没有函数直接执行 I/O 操作，因此进程从不阻塞。由于没有任何阻塞，可伸缩系统在 Node 中开发是非常合理的。
+这与当今比较常见的采用操作系统线程的并发模型形成了鲜明对比。基于线程的网络效率相对较低且更难以使用。此外，由于没有锁，Node.js 的用户不用担心进程死锁的问题。Node.js 中几乎没有函数直接执行 I/O 操作，其进程从不会被阻塞，因此用 Node.js 来开发可扩展系统是非常合理的。
```

Comment:
> 此外，由于没有锁，Node.js 的用户不用担心进程死锁的问题

IMO we need to emphasize that this is in user-land to avoid other ambiguities.

Reply:
I strongly suggest you submitting it into the "Issues" of the doc, this is a discussion instead of suggestions for the translation. Because if you do this, the English version will be first changed.

I think you can directly do changes to the English version and let tech members have a review first, and then we can translate them into Chinese, and call others to related languages as well ;)

Reply:
> > 此外，由于没有锁，Node.js 的用户不用担心进程死锁的问题
> 
> IMO we need to emphasize that this is in user-land to avoid other ambiguities.

I strongly suggest you submitting it into the "Issues" of the doc, this is a discussion instead of suggestions for the translation. Because if you do this, the English version will be first changed.

I think you can directly do changes to the English version and let tech members have a review first, and then we can translate them into Chinese, and call others to related languages as well ;)

Reply:
> Furthermore, users of Node.js are free from worries of dead-locking the process, since there are no locks.

Hmm, the English version is OK because it mentioned the user at the former sentence, however, the Chinese description in this PR doesn't guarantee this.